library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -78,6 +80,24 @@ public interface ResourceLoader {
 	 */
 	void registerResourcePackProfileProvider(@NotNull ResourcePackProvider provider);
 
+	/**
+	 * {@return the registration of default resource packs event}
+	 * <p>
+	 * This event is triggered when the default resources are created and allow to register additional resource packs
+	 * that are displayed to the user as the {@code "Default"} resource pack.
```

Comment:
Does this mean that the resource packs are merged into the "Default" resource pack?

Reply:
Yes, they are merged into that resource pack as it simplifies a ton their injection, they are merged in a way they have more priority than Vanilla but less priority than modded resource packs from disk.